src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -1006,10 +933,21 @@ void program_node::init_onednn_primitive_attributes() {
                                   type == onednn_post_op_type::binary_relu ||
                                   type == onednn_post_op_type::scale ||
                                   type == onednn_post_op_type::sum;
+
         if (has_memory_buffers)
             memory_offset++;
     };
 
+    auto cant_use_output_scales = [&](size_t idx, data_types in_data_type, layout in_layout) {
+        return (idx != 0 ||
```

Comment:
In onednn 2.7, output_scale was applicable only when idx == 0.
However, it is applicable only when idx is the last.

Reply:
ACC issue